databases/gcp-mysql-ssl/README.md

```diff
@@ -0,0 +1,32 @@
+# GCP MySQL with SSL
+
+Google Cloud MySQL with SSL
+
+Prisma docs: https://www.prisma.io/docs/concepts/database-connectors/mysql#configuring-an-ssl-connection
+GCP docs: https://cloud.google.com/sql/docs/mysql/configure-ssl-instance
+
+## How to run this locally
+
+### Environment variables
+
+The environment variable `GCP_MYSQL_SSL_DB_URL` should point to a GCP MySQL database. Download the certificates based on GCP docs and put them in the root directory next to this `README.md` file.
+
+The connection string `GCP_MYSQL_SSL_DB_URL` would look like:
+
+`mysql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`
+
+The following files are expected:
+
+- server-ca.pem
+- client-cert.pem
+- client-key.pem
+
```

Comment:
Then maybe something like this:

```suggestion
The environment variable `GCP_MYSQL_SSL_DB_URL` should point to a GCP MySQL database. It should follow the following pattern:

`mysql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`

The 3 mentioned files `server-ca.pem`, `client-cert.pem`, `client-key.pem` are to be downloaded from GCP and put into the root directory next to this `README.md` file. (In the tests of this project, these files are created on demand from environment variables that contain their content)
```

Problem: The connection string only mentions 2 fils, and only 1 is listed in the list of files :shrug:

Reply:
Ah yes, the docs mention that and that is why I linked the docs I can copy that part over here but that would be duplicated docs, possibility of being outdated etc 🤷 

This is the line from the docs that produces the other file using the remaining listed files

`openssl pkcs12 -export -out client-identity.p12 -inkey client-key.pem -in client-cert.pem`

Reply:
An alternative can be to tell them to fill all the env vars i.e. load their certs in env vars and then just run `sh run.sh; sh test.sh` like the CI does but I feel that is clunkier (specifically loading certs in env vars bit).

Reply:
Changed the readme so this `openssl` command is used in `sh run.sh`. Making the env vars optional

Reply:
Waiting for CI to be green.

Reply:
And it is, replicated the same in the Postgres test.